blocks/library/html/index.js

```diff
@@ -70,6 +71,6 @@ export const settings = {
 	] ),
 
 	save( { attributes } ) {
-		return attributes.content;
+		return <DangerousHTML>{ attributes.content }</DangerousHTML>;
```

Comment:
I think this component should be called `RawHTML`, it doesn't make a lot of sense to call it dangerous in the context of an editor that deals with raw html in valid and regular cases.

Reply:
Works for me.

Reply:
Yeah, one potential downside is it's basically a substitute for `dangerouslySetInnerHTML` (sans wrapper), not just for serialization, so it could turn into a bit of a foot-gun.